pkg/neonvm/controllers/vm_controller.go

```diff
@@ -1178,6 +1212,10 @@ func imageForVmRunner() (string, error) {
 	return image, nil
 }
 
+// podSpec returns a VirtualMachine Pod object
+// withReadinessProbe - if true, adds a readiness probe to the container
```

Comment:
question: What does `withReadinessProbe` refer to?

Reply:
leftover from previous iteration, forget to clean up, thanks for pointing out